var/spack/repos/builtin/packages/py-pymongo/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPymongo(PythonPackage):
+    """The PyMongo distribution contains tools for interacting with
+    MongoDB database from Python. The bson package is an implementation
+    of the BSON format for Python. The pymongo package is a native
+    Python driver for MongoDB. The gridfs package is a gridfs
+    implementation on top of pymongo.
+
+PyMongo supports MongoDB 2.6, 3.0, 3.2, 3.4, 3.6, 4.0 and 4.2."""
```

Comment:
Indentation is hurting my eyes! :laughing:

Reply:
> Indentation is hurting my eyes!

Fixed!